templates/components/settings/python/settings.py

```diff
@@ -1,9 +1,24 @@
 import os
-from typing import Dict
+from typing import Dict, Optional
 
+from llama_index.core.multi_modal_llms import MultiModalLLM
 from llama_index.core.settings import Settings
 
 
+def get_multi_modal_llm() -> Optional[MultiModalLLM]:
+    model_provider = os.getenv("MODEL_PROVIDER")
+    llm_model = os.getenv("MULTIMODAL_LLM_MODEL")
```

Comment:
we still should use the model env variable and return null if it's not multimodal